src/components/MenuItem.js

```diff
@@ -150,8 +153,18 @@ const MenuItem = (props) => {
                                 </Text>
                             </View>
                         )}
+                        {!_.isEmpty(props.floatRightAvatars) && (
+                            <View style={[styles.justifyContentCenter, (props.brickRoadIndicator ? styles.mr4 : styles.mr3)]}>
+                                <MultipleAvatars
+                                    icons={props.floatRightAvatars}
+                                    size={props.viewMode === CONST.OPTION_MODE.COMPACT ? CONST.AVATAR_SIZE.SMALL : CONST.AVATAR_SIZE.DEFAULT}
+                                    fallbackIcon={Expensicons.Workspace}
+                                    shouldStackHorizontally={props.shouldStackHorizontally}
+                                />
+                            </View>
+                        )}
                         {Boolean(props.brickRoadIndicator) && (
-                            <View style={[styles.alignItemsCenter, styles.justifyContentCenter]}>
+                            <View style={[styles.alignItemsCenter, styles.justifyContentCenter, styles.l4]}>
```

Comment:
We had a small inconsistent padding issue caused by this change. 

Issue: https://github.com/Expensify/App/issues/18418
Issue Tittle: Workspaces icons and green dot has inconsistent padding